rastergm.i

```diff
@@ -10,12 +10,12 @@
         $action
     } catch (const std::exception &e) {
         SWIG_exception(SWIG_RuntimeError, e.what());
+    } catch (const csm::Warning &e) {
+        PyErr_WarnEx(PyExc_UserWarning, e.getMessage().c_str(), 1);
     }
```

Comment:
Takes the csm::Warning and propagates as a python UserWarning. The message is the message that the csm::Warning is putting out.